crypto/armor.go

```diff
@@ -29,6 +31,19 @@ const (
 	headerType    = "type"
 )
 
+var (
+	kdf       = "kdf"
```

Comment:
```suggestion
	headerKdf       = "kdf"
```

nit: just for consistency with headerType and headerVersion